Add dropdown types

boutinb opened the pull request.

boutinb committed:
Start

boutinb committed:
more

boutinb committed:
more

boutinb committed:
Update jaspColumnEncoder

JorisGoosen committed:
some tweaks for icons and removed suggested columns

also use jaspColumnEncoder from EnumAsVector branch

JorisGoosen committed:
rename transparent to transformed and make sure that older inactive symbols are used in variableslist

JorisGoosen committed:
icoontjes

JorisGoosen committed:
better icons and make data icons correct again

JorisGoosen commented:
So @boutinb ive added the icons and the totalNumerics path for variableslist.
I think we should probably still have:
- properties: `minNumericLevels`, `maxNumericLevels`.
- And perhaps: `minLevels` and `maxLevels`

These both can then be the check for the drag->drop, instead of the type.

boutinb committed:
show transformed icon if different type

boutinb committed:
add a preview of the transformed type

boutinb committed:
Do not check type in VariablesList

The type is then automatically set to the VariablesList (default) type.
The types are also given to the options with the extra array entry: "<option name>.types".

boutinb committed:
Add min/max levels constraints in VariablesList

boutinb committed:
Add variables types in options

JorisGoosen commented:
Ok awesome!
